src/nwb_datajoint/common/common_waveforms.py

```diff
@@ -54,6 +57,10 @@ def make(self, key):
         key['waveform_extractor_path'] = str(Path(os.environ['NWB_DATAJOINT_WAVEFORMS_DIR']) / Path(waveform_extractor_name))
         if os.path.exists(key['waveform_extractor_path']):
             shutil.rmtree(key['waveform_extractor_path'])
+        if waveform_params['whiten']:
+            recording = toolkit.preprocessing.whiten(recording)
+            # remove the 'whiten' dictionary entry as it is not recognized by spike interface
+            del waveform_params['whiten']
```

Comment:
what if `waveform_params['whiten']` is `False`? still would need to remove `waveform_params['whiten']`